gradunwarp/core/gradient_unwarp.py

```diff
@@ -113,12 +112,3 @@ def run(self):
     def write(self):
         self.unwarper.write(self.args.outfile)
 
-
-if __name__ == '__main__':
```

Comment:
This deletion looks like it would fail if anything tried to run it via the old command.  If we add support to run it as another name, fine, but making it error in scripts that used to work is unfriendly.